README.md

```diff
@@ -20,7 +20,6 @@ It currently includes:
 * [Circe] and http4s integration
 * [Decline Effect]
 * [Munit Cats Effect]
-* [fs2-data-csv]
```

Comment:
Wasn't this the line you added in another PR? :D

Reply:
Yes that's me haha!
But somehow a second fs2-data mention got sneaked in:
![image](https://user-images.githubusercontent.com/1857826/224503775-a90d72a8-932e-4414-b879-880bbec42755.png)

Reply:
lol I've just realized it's a duplicate